pkg/runtime/adoption_reconciler.go

```diff
@@ -224,10 +224,16 @@ func (r *adoptionReconciler) Sync(
 	}, described.RuntimeObject()); err != nil {
 		if apierrors.IsNotFound(err) {
 			// If Adopted AWS resource was not found in k8s, create it.
+
+			// Before creation, Keep the copy of original described object
+			// because after the create call, Status gets set to empty
+			describedCopy := described.DeepCopy()
```

Comment:
How can a described object have a non empty .Status but does not exist in the k8s cluster?

Reply:
If the custom resource for adopted resource does not exist in K8s cluster, the create call is made. Since we pass the `described` object in the create call, it's status gets updated to empty to match etcd copy of custom resource.

Reply:
Ohhh nvm i didn't realize this was the adoption controller